src/backend/main/py.main/caliopen_main/user/core/user.py

```diff
@@ -185,12 +185,13 @@ def create(cls, new_user):
         # then
         #      create user and linked contact
         """
+
         def rollback_username_storage(username):
             UserName.get(username).delete()
 
         # 0. check for user email white list and max number of users
-        cls._check_whitelistes(new_user)
-        cls._check_max_users()
+        # cls._check_whitelistes(new_user)
```

Comment:
uncomment that 2 lines that must still active

Reply:
=> commit #68c73ee